### DISCUSSION TITLE
Are Babel Plugins supported yet? How can I use them?

### DISCUSSION THREAD

**Discussion by jasikpark** [swc-project/swc#1904] [Q&A] [answered]
I'd like to use `swc` to run some babel transforms more quickly than babel can, but I'm unsure if `swc` has support for babel plugins right now?

**Comment by FrederickEngelhardt** [comment]
Bump. The docs say they are supported but how do you use them? Attempting to port metro's babel config over. Hoping to see some examples of integration with babel-transforms.

Wish the docs here had a link to how to integrate the transforms. https://swc.rs/docs/comparison-babel

**Reply by kdy1** [reply, marked as answer]
It's not supported. Can you point me a doc which says it's supported?
